homework_5/Methods/FindSymbolOccurance.java

```diff
@@ -1,5 +1,24 @@
 package Methods;
 
 
+import java.util.Scanner;
+
 public class FindSymbolOccurance {
+    public static int findSymbolOccurance(String source, char letter){
```

Comment:
What will happen, if source string == null?
Please add null validation only for source string